src/map/party.cpp

```diff
@@ -200,10 +200,18 @@ uint8 CParty::MemberCount(uint16 ZoneID)
 
     for (uint32 i = 0; i < members.size(); ++i)
     {
-        if (members.at(i)->getZone() == ZoneID)
+        auto* member = members.at(i);
+        if (member->getZone() == ZoneID)
         {
             count++;
         }
+        if (member->objtype == TYPE_PC)
+        {
+            for (auto* trust : static_cast<CCharEntity*>(member)->PTrusts)
+            {
+                count++;
```

Comment:
I decided not to treat trusts as full party members (they don't exist inside `members` and they don't return with `getParty()`). This might change in the future, but it'll take a lot more plumbing and loads of testing.